src/manifest.ts

```diff
@@ -63,6 +65,7 @@ interface Package {
   bumpMinorPreMajor?: boolean;
   changelogSections?: ChangelogSection[];
   changelogPath?: string;
+  releaseAs?: string;
```

Comment:
For `release-as` I'm guessing a user would most likely use this during a manual run of the `relese-please` CLI?